### PR TITLE
setup react jsx

### PR DESCRIPTION
**Pull request by Ntare22**
create components for the numbers
create component for the screen

### CODE REVIEW

File: src/components/NumsOperations.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  { key: 'three', value: 3 },
+  { key: 'plus', value: '+' },
+  { key: 'zero', value: 0 },
+  { key: 'dot', value: '.' },
+  { key: 'equals', value: '=' },
+];
+
+// eslint-disable-next-line react/prop-types
+const NumsOperations = ({ items }) => {
+  // eslint-disable-next-line react/prop-types
```
Comment (bimbolabuari) [inline src/components/NumsOperations.js]: Kindly remove this disabled linter, linting improves readability, removes errors. 👍

Please check for other occurrences 🙂

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import NumsOperations from './NumsOperations';
+
+export default function Calculator() {
+  return (
+    <>
+      <CalcScreen number="0" />
+      <NumsOperations />
+    </>
+  );
+}
```
Comment (bimbolabuari) [inline src/components/Calculator.js]: As required, Kindly use class based components for the files 🙂

File: src/App.js [hunk 3]
```diff
@@ -1,29 +0,0 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
```
Comment (bimbolabuari) [inline src/App.js]: According to the requirements, Let's import the calculator.js and use it in your main component (App.js). 👍

Kindly use a class component also 😉

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hello @Ntare22 , 

You have done a good job so far 🚀  

## Highlights 🎉 
- Deleted all the boilerplate from CRA (text, images, styles). 🚀 
- Created a directory called components. 💯 
- Inside components, created a new Calculator.js file. 🥇 


There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/index.js [hunk 4]
```diff
@@ -1,17 +1,11 @@
...
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
   document.getElementById('root'),
 );
```
Comment (rukundoeric) [inline src/index.js]: - [ ] <b>REQUIRED:</b> As mentioned in the previous review, `Calculator` should not be imported and used in here, it should be imported and used in `App` component. Kindly create `App` component in `src/` directory and then import and use `Calculator` component in there. Furthermore, please make sure that the `App` component is also class-based component.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Ntare22, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - N/A

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @Ntare22 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**